generator/src/routes/+page.svelte

```diff
@@ -61,33 +140,68 @@
         </div>
       </div>
       <div class="actions-wrapper">
-        <Button title="Generate IOXIO Tag" onClick={() => {}} />
+        <Button disabled={status === Status.GENERATING} title="Generate IOXIO Tag" type="submit" />
       </div>
     </form>
   </div>
   <!-- Right Panel -->
   <div class="qrcode-area-wrapper">
     <div class="qrcode-area">
-      <div class="qrcode">
-        <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+      <div class="qrcode-frame">
+        {#if status !== Status.GENERATING}
+          <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+          <img class="logo" src={LogoTagsSvg} alt="" aria-hidden="true" />
+          {#if status === Status.FINISHED}
+            <img class="qrcode" alt="" aria-hidden="true" src="" bind:this={qrcodeElement} />
+            <div class="ioxio-tag-frame">
+              <img class="tag" src={LogoTagsBlackSvg} alt="" aria-hidden="true" />
+            </div>
+          {/if}
+        {:else}
+          <div class="frame anim" />
+        {/if}
         <img class="effect" src={EffectSvg} alt="" aria-hidden="true" />
-        <img class="logo" src={LogoTagsSvg} alt="" aria-hidden="true" />
       </div>
     </div>
-    <div class="status">IOXIO Tag generator</div>
+    <div class="status">
+      {#if status === Status.READY}
+        IOXIO Tag generator
+      {:else if status === Status.GENERATING}
+        Generating...
+      {:else}
+        Finished!
+      {/if}
+    </div>
     <div class="product-area">
-      <div class="description">
-        <p>
-          This application serves as a demonstration platform for generating QR codes specifically
-          designed for Digital Product Passports.
-        </p>
-        <p>
-          Utilizing the IOXIO Dataspace technology, this app showcases how QR codes can be generated
-          to serve as digital passports for various Data Products. These QR codes can be read using
-          a compatible reader application, enabling access to relevant product details in real time.
-        </p>
-        <a class="documentation" href="/">See documentation →</a>
-      </div>
+      {#if status === Status.READY}
+        <div class="description">
+          <p>
+            This application serves as a demonstration platform for generating QR codes specifically
+            designed for Digital Product Passports.
+          </p>
+          <p>
+            Utilizing the IOXIO Dataspace technology, this app showcases how QR codes can be
+            generated to serve as digital passports for various Data Products. These QR codes can be
+            read using a compatible reader application, enabling access to relevant product details
+            in real time.
+          </p>
+          <a class="documentation" href="/">See documentation →</a>
```

Comment:
Please put `documentationUrl` in the configuration, there's now multiple links to `/` and finding them later is going to be hard